src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,104 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final String INDEX_EXCEPTION_MESSAGE = "Element missing for given index ";
+    private static final String AVAILABILITY_EXCEPTION_MESSAGE = "Element missing: ";
```

Comment:
Technically you can do it but in real projects exception messages don’t  make constant